docs/setup_guides/cloud.rst

```diff
@@ -115,28 +116,20 @@ so you can use it directly.
 
   set -e
 
-  PROJECTID=<YOUR GOOGLE CLOUD PROJECT ID>
-  # this is the zone used by pangeo.pydata.org
-  ZONE='us-central1-b'
+  echo $PROJECTID
 
-  # cluster size settings: modify as needed to fit your needs / budget
   NUM_NODES=2
-  MIN_WORKER_NODES=0
-  MAX_WORKER_NODES=100
-  CLUSTER_NAME='pangeo'
-
-  # https://cloud.google.com/compute/pricing
-  # change the machine typer based on your computing needs
-  WORKER_MACHINE_TYPE='n1-standard-4'
 
   # create cluster on GCP
-  gcloud config set container/new_scopes_behavior true
   gcloud config set project $PROJECTID
   gcloud container clusters create $CLUSTER_NAME --num-nodes=$NUM_NODES --zone=$ZONE \
       --machine-type=n1-standard-2 --no-enable-legacy-authorization
+  #gcloud container node-pools create worker-pool --zone=$ZONE --cluster=$CLUSTER_NAME \
+  #    --machine-type=$WORKER_MACHINE_TYPE --preemptible --enable-autoscaling \
+  #    --num-nodes=$MIN_WORKER_NODES --max-nodes=$MAX_WORKER_NODES --min-nodes=$MIN_WORKER_NODES
   gcloud container node-pools create worker-pool --zone=$ZONE --cluster=$CLUSTER_NAME \
-      --machine-type=$WORKER_MACHINE_TYPE --preemptible --enable-autoscaling \
-      --num-nodes=$MIN_WORKER_NODES --max-nodes=$MAX_WORKER_NODES --min-nodes=$MIN_WORKER_NODES
+      --machine-type=$WORKER_MACHINE_TYPE --preemptible --num-nodes=$MIN_WORKER_NODES
+  gcloud container clusters update $CLUSTER_NAME --zone=$ZONE --node-pool=worker-pool --enable-autoscaling --max-nodes=$MAX_WORKER_NODES --min-nodes=$MIN_WORKER_NODES
```

Comment:
Thos two lines where needed only because of the GCE problem with container API, I've not update the scripts...

I think your right about trying to synchronise scripts from the repo and this page. We should probably remove every script snippets from this page.